lobby/src/main/java/org/triplea/lobby/server/db/UserDao.java

```diff
@@ -26,4 +26,6 @@
   boolean login(String username, HashedPassword password);
 
   DBUser getUserByName(String username);
+
+  void makeAdmin(DBUser dbUser);
```

Comment:
We should really hook this up soon-ish. Currently it's only being used by test-code.

Reply:
Thanks for calling this out. It might get deleted in the meantime until we have a use for it. The schema could change completely, keeping something like this along is only adding overhead. That level of coupling to dead code makes changing things much slower. 

I'm not sure if it's better to just remove this for now, but in the meantime we need it part of the API to avoid using the implementation. Let's keep an eye on this and see how things evolve. Allowing more time for a decision will give more information and allow for a better decision.